src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.tsx

```diff
@@ -176,4 +176,33 @@ const TagItem = styled('div')`
   width: 50%;
 `;
 
+const TagBarGlobalSelectionLink = styled(GlobalSelectionLink)`
+  position: relative;
+  display: flex;
+  line-height: 2.2;
+  color: #493e54;
+  margin-bottom: ${space(0.5)};
+  padding: 0 ${space(1)};
+`;
+
+const TagBarBackground = styled('div')`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  background: #f6f7f8;
```

Comment:
theme color?

Reply:
yes

Reply:
I would normalize on something in theme. There are some chart related colours that map to grey200 in lightmode that could work here.